topics/server-requests.md

```
@@ -65,17 +81,49 @@ example, `call.parameters["login"]` in the code snippet below will return _admin
+
+The following example accesses the `price` query parameter from a request made to `/products?price=asc`:
 
 ```kotlin
 ```
 {src="snippets/_misc/QueryParameter.kt"}
 
 You can also get the entire query string using the [`ApplicationRequest.queryString()`](https://api.ktor.io/ktor-server-core/io.ktor.server.request/query-string.html) function.
 
+## Required request parameters
+
+When handling requests, it is common to extract values from [path parameters](#path_parameters),
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)